packages/flutter_snippets/0.1.0/_manifest.yml

```diff
@@ -0,0 +1,5 @@
+name: 'flutter-snippets'
+title: 'Flutter snippets'
+description: A package for flutter code snippets
+version: 0.1.0
+author: rishi
```

Comment:
Some `tags:` would enable [Espanso Hub](https://hub.espanso.org/search) to categorise the package, e.g. "Development", "Flutter", please.